src/pages/resultsView/mutation/ResultsViewMutationMapperStore.ts

```diff
@@ -82,7 +95,25 @@ export default class ResultsViewMutationMapperStore extends MutationMapperStore
     ) {
         super(
             mutationMapperConfig,
-            mutationMapperStoreConfig,
+            (() => {
+                mutationMapperStoreConfig['filterAppliersOverride']![
+                    MutationTableColumnType.GNOMAD
+                ] = createNumericalFilter((d: Mutation) =>
+                    GnomadColumnFormatter.getSortValue(
+                        [d],
+                        this.indexedMyVariantInfoAnnotations
+                    )
+                );
+                mutationMapperStoreConfig['filterAppliersOverride']![
+                    MutationTableColumnType.DBSNP
+                ] = createCategoricalFilter((d: Mutation) =>
+                    DbsnpColumnFormatter.download(
+                        [d],
+                        this.indexedMyVariantInfoAnnotations
+                    )
+                );
+                return mutationMapperStoreConfig;
+            })(),
```

Comment:
Unfortunately, I need to access `this.indexedMyVariantInfoAnnotations` so I can't access it from a `static` method; nor can I pass it to `mutationMapperStoreConfig` from `ResultsViewPageStore` since it's a property of `DefaultMutationMapperStore`.

Therefore I have to either leave my function as is or add those two filter appliers to `@computed get numericalFilterColumns` and `@computed get categoricalFilterColumns` respectively, which is prettier but a bit less efficient.